README.md

```diff
@@ -21,6 +21,36 @@ pip install .[all]  # Install the remaining elements of this package.
 
 Aside from the base packages, you can optionally install `dev` tools using this tag, `materials` dependencies using this tag, or `molecules` packages using this tag. The simplest option is to use the `all` tag, as above, which installs all dependencies.
 
+## Main Components of the GFlowNet Library
+
+The GFlowNet library has 5 core components, each playing a crucial role in the network's operation. Understanding these components is essential for effectively using and extending the library for your own task. These 5 component are the Logger, Proxy, Environment,  Policies (Forward and Backward), and GFlowNet Agent. 
```

Comment:
I would describe the components in the following order: Environment, Proxy, Policies and Agent.